anitacheung83/Timetable-Wallpaper
# Feature Added: Text Color Option

**Feature Added**: Text Color Option

**Description**:
This pull request introduces a new feature that allows users to select text colour in the timetable. Users can now choose black or white text colour to personalize their experience.

**Changes Made**:
* Added a text colour selector in the settings
* Integrated the selected colour with the Timetable

**How to test**:
1. Open the web app and navigate to the settings menu.
2. You will find a new "Text Color" option.
3. Choose a color from the avaliable options
4. Observe the text color changes in the application to reflect your selection.

**Screenshots**
<img width="414" alt="Screenshot 2023-10-16 at 5 04 05 PM" src="https://github.com/anitacheung83/Timetable-Wallpaper/assets/77123829/ef315720-bfe6-491f-91d3-ec1cd1214f0a">

**Checklist**:
- [X] Code has been tested locally and works as expected
- [X] Unit tests have been added or updated to cover the new feature
- [ ] Documentation has been updated to reflect the new option in the user guide
